app/webpacker/styles/application.scss

```diff
@@ -40,6 +40,10 @@ $govuk-global-styles: true;
   background-color: #0076bd;
 }
 
+.govuk-hint  {
```

Comment:
I don't think this is needed if form builder is also bumped to 3.0.0.

Reply:
That worked thanks 👍